yarn.lock

```diff
@@ -6806,13 +6822,20 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@typescript-eslint/types@npm:8.35.1, @typescript-eslint/types@npm:^8.34.0, @typescript-eslint/types@npm:^8.35.0, @typescript-eslint/types@npm:^8.35.1":
+"@typescript-eslint/types@npm:8.35.1":
```

Comment:
Note to self: upgrade `typescript-eslint` once this lands to dedupe this dependency.